packages/scripts/README.md

```diff
@@ -125,6 +143,23 @@ _Example:_
 This is how you execute the script with presented setup:
 * `npm run lint:css` - lints CSS files in the whole project's directory.
 
+### `start`
+
+Builds the code for development to the `build` folder. The script will automatically rebuild if you make changes to the code. You will see the build errors in the console. It uses [Webpack](https://webpack.js.org/) behind the scenes and you still need to provide your own config as described in the [documentation](https://webpack.js.org/concepts/configuration/).
```

Comment:
> Builds the code for development to the `build` folder. 

(here and build section of the documentation)

Could this be clarified?  It will build to whatever is configured in the webpack configuration right?  So that folder may not be the "build" folder.  Maybe something like:

> Builds the code for development to the designated `build` folder in your webpack configuration file.

Reply:
Thanks, I will clarify as suggested 👍

Reply:
I went with:

> Builds the code to the designated `build` folder in the configuration file.

I didn't want to be very specific about Webpack and who has to create the config. I hope we can provide the default config very soon in a follow-up PR.